Dockerfile

```diff
@@ -1,5 +1,5 @@
 ARG OPENVDS_IMAGE=openvds
-ARG VDSSLICE_BASEIMAGE=golang:1.20-alpine3.16
+ARG VDSSLICE_BASEIMAGE=golang:1.20-alpine3.17
```

Comment:
Are you happy with the bump to Alpine 3.17 or would you consider to bump even further, e.g., 3.19?

Reply:
I also just realised that Go 1.22 is out which means that Go 1.20 is out of support. We could consider to upgrade to Go 1.22. It may give some better performance as well, see [Go 1.22 release notes](https://go.dev/doc/go1.22). I will try to build a Docker image with Go 1.22 locally and see if that shows any immediate blockers.